src/gaxios.ts

```diff
@@ -404,6 +404,7 @@ export class Gaxios {
       return data as {};
     } else if (
       contentType.includes('text/plain') ||
+      contentType.includes('text/csv') ||
```

Comment:
Now I'm just thinking if we should instead do `contentType.startsWith('text/')` and consider all text content the same. @sofisl what do you think?

Reply:
I agree! Would you mind including a regex instead? and maybe a `.match`? Thank you!